tests/python/gaia-ui-tests/gaiatest/tests/functional/bugzillalite/manifest.ini

```diff
@@ -0,0 +1,7 @@
+[parent:../manifest.ini]
+
+[DEFAULT]
+external = true
```

Comment:
external = true by default, see: http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/manifest.ini#54
So this is not really necessary, it seems.